### PR TITLE
qimgv-video: Add version 0.9.1

### PR DESCRIPTION
**Pull request by bradenhilton**

### CODE REVIEW

File: bucket/qimgv-video.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+    "version": "0.9.1",
+    "description": "Qt5 image viewer",
```
Comment (Ash258) [inline bucket/qimgv-video.json:3]:
```suggestion
    "description": "Fast, configurable and easy to use image viewer with video support",
```

File: bucket/qimgv-video.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    "extract_dir": "qimgv",
+    "bin": "qimgv.exe",
+    "shortcuts": [
+        [
+            "qimgv.exe",
+            "qimgv"
+        ]
+    ],
+    "persist": [
+        "conf",
```
Comment (Ash258) [inline bucket/qimgv-video.json:25]: I would persist `cache` also

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/qimgv-video.json (hunk 2 above)
Comment (bradenhilton) [reply bucket/qimgv-video.json:25]: That's fine with me, I will implement the changes now, but the non-video manifest does not do this.

**Review by Ash258 — APPROVED** [review]

**Pull request merged**